### ISSUE TITLE
Remove feature to assign different service accounts based on admin status

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/infrastructure#4818]

**Comment by Gman0909** [comment]
I've added the feature to the product roadmap, but would like an update from @yuvipanda as to whether this task, of removing the half finished implementation, still needs doing.

**Comment by yuvipanda** [comment]
It does need doing!

**Comment by sgibson91** [comment]
A couple of clarifying questions here since, as noted, this is a half-baked feature so the knowledge is not distributed across the team.

1. > Check to see if the community we deployed it for is actually using this feature

   Who is the community? I'm guessing from the mention of Ryan Abernathy in the linked issue copied to product board that is linked to from a Slack thread linked in this issue that the community may be LEAP. Can we confirm and propagate that info into this issue so that whoever picks it up doesn't have to go down the same rabbit hole I just did? There is also a mention that we support this on GCP and AWS - who (if anyone) is the likely community on AWS?

2. Can we have links to at least the files that contain the code that needs removing?

With these two points answered, I would consider this issue to be _truly_ refined and easily picked up by anyone.

**Comment by sgibson91** [comment]
AWS community _may_ have been M2LInES? I can't remember if they were AWS or GCP, and they've been decommissioned now anyway.

**Comment by yuvipanda** [comment]
https://github.com/2i2c-org/infrastructure/pull/3952 was the PR partially adding this feature. Digging through that, I see that opensci is the community this was enabled for. And it was used to give admins ability to upload files to the `[opensci-persistent-sciencecore](https://us-west-2.console.aws.amazon.com/s3/buckets/opensci-persistent-sciencecore)` s3 bucket. I see files in there mostly from April 16 - the same day the PR was deployed, and untouched since.

So I think the confirmation is to ask @jmunroe (original requestor of this feature) wether this can be removed. I'll update the body

**Comment by yuvipanda** [comment]
Thanks for spotting what was missing, @sgibson91. I've updated the description and it's back to refined now I think.

**Comment by jmunroe** [comment]
I agree this looks like it was a one-off solution to a need that a community needed in April 2024.  I concur with removing the feature if it not part of our standard offerings.

If and when there is demand for such a feature it rightly should be go through the product process.
